elements/pfe-dropdown/src/pfe-dropdown.scss

```diff
@@ -0,0 +1,192 @@
+  ),
+  toggle-text: (
+    Overflow: hidden,
+    TextOverflow: ellipsis,
+    WhiteSpace: nowrap
+  ),
+  toggle-icon: (
+    VerticalAlign: calc(#{pfe-var(container-spacer)} * -0.125),
+    Height: #{pfe-var(icon-size)},
+    Width: #{pfe-var(icon-size)},
+    MarginRight: calc(#{pfe-var(container-spacer)} * 0.5),
+    MarginLeft: #{pfe-var(container-spacer)},
+    LineHeight: #{pfe-var(line-height)}
+  ),
+  menu: (
+    Top: calc(100% + 0.25rem),
+    ZIndex: 200,
+    PaddingTop: calc(#{pfe-var(container-padding)} * 0.5),
+    PaddingBottom: calc(#{pfe-var(container-padding)} * 0.5),
+    PaddingRight: 0,
+    PaddingLeft: 0,
+    Margin: 0,
+    Background: #{pfe-color(surface--lightest)},
+    BackgroundClip: padding-box,
+    BorderWidth: #{pfe-var(ui--border-width)},
+    BorderStyle: #{pfe-var(ui--border-style)},
+    BorderColor: transparent,
+    BoxShadow: #{pfe-var(box-shadow--sm)},
+    BoxSizing: border-box,
+    ListStyle: none,
+    MinWidth: 100%
+  )
+);
+
+:host {
+  @include pfe-print-variables($variables);
+  position: relative;
+  display: inline-block;
+  max-width: 100%;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([is_disabled]) {
+  .pfe-dropdown {
+    &__container {
+      pointer-events: none;
+    }
+    &__toggle {
+      --pfe-dropdown__toggle--BackgroundColor: #ededed;
+      &::before {
+        border: none;
+      }
+    }
+  }
+}
+
+.pfe-dropdown {
+  &__toggle {
+    position: #{pfe-local($cssvar: Position, $region: toggle)};
+    display: #{pfe-local($cssvar: Display, $region: toggle)};
+    align-items: #{pfe-local($cssvar: AlignItems, $region: toggle)};
+    justify-content: #{pfe-local($cssvar: JustifyContent, $region: toggle)};
+    min-width: #{pfe-local($cssvar: MinWidth, $region: toggle)};
+    max-width: #{pfe-local($cssvar: MaxWidth, $region: toggle)};
+    padding-top: #{pfe-local($cssvar: PaddingTop, $region: toggle)};
+    padding-right: #{pfe-local($cssvar: PaddingRight, $region: toggle)};
+    padding-bottom: #{pfe-local($cssvar: PaddingBottom, $region: toggle)};
+    padding-left: #{pfe-local($cssvar: PaddingLeft, $region: toggle)};
+    font-size: #{pfe-local($cssvar: FontSize, $region: toggle)};
+    font-weight: #{pfe-local($cssvar: FontWeight, $region: toggle)};
+    line-height: #{pfe-local($cssvar: LineHeight, $region: toggle)};
+    color: #{pfe-local($cssvar: Color, $region: toggle)};
+    background-color: #{pfe-local($cssvar: BackgroundColor, $region: toggle)};
+    border: #{pfe-local($cssvar: Border, $region: toggle)};
+    font-family: inherit;
+    &:hover,
+    &:active,
+    &:focus {
+      &:before {
+        border-bottom-color: #{pfe-local(
+            $cssvar: BorderBottomColor--before--hover,
+            $region: toggle
+          )};
+      }
+    }
+    &:active,
+    &:focus {
+      &:before {
+        border-width: #{pfe-local(
+            $cssvar: BorderWidth--before--hover,
+            $region: toggle
+          )};
+      }
+    }
```

Comment:
@starryeyez024 I say leave it for accessibility. @samlinville Thoughts?

Reply:
@coreyvickery  i was thinking that `:focus` would still serve the accessibility needs, and then once it's open `.active` doesn't need a glow around it because its already visibly open, and has the blue border on the bottom.

Reply:
@starryeyez024 @coreyvickery I don't see a blue outline when I use the dropdown demo on the PF4 site here https://www.patternfly.org/v4/documentation/react/components/dropdown, in Firefox, so not sure where that's popping up

Generally, agree that we do not need the glow on `:active` since `:focus` is the more accessibility-focused state. However, in cases like this where we're implementing a PF4 React component as a PFE web component, I really think we should mimic design exactly and push any design changes through the main PatternFly project so that there's consistency between the PF4 React and PFE. As a designer I do not want to have to chase down small differences between components– we'll end up in a world where designers have to dictate whether to use the PF4 React component vs. the PFE component because there are design differences that matter to us.

Reply:
@samlinville sure -- do you want to take this suggested revision back to the design workflow, and we'll just keep this as-is for now?

Reply:
Sure, I can write up a ticket for it. Thanks! @starryeyez024